src/Parser/VengeanceDemonHunter/CombatLogParser.js

```diff
@@ -14,6 +16,7 @@ import getCastEfficiency from 'Parser/Core/getCastEfficiency';
 
 import ISSUE_IMPORTANCE from 'Parser/Core/ISSUE_IMPORTANCE';
 
+import AbilityTracker from 'Parser/Core/Modules/AbilityTracker';
```

Comment:
This is unnecessary since this is already added in the `defaultModules` of the core CombatLogParser (which you extend)